cmd/minikube/cmd/start.go

```diff
@@ -1424,6 +1431,28 @@ func validateDiskSize(diskSize string) error {
 	return nil
 }
 
+// validateWindowsOSVersion validates the supplied window server os version
+func validateWindowsOSVersion(osVersion string) error {
+	validOptions := node.ValidWindowsOSVersions()
+
+	if osVersion == constants.DefaultWindowsNodeVersion {
+		return nil
+	}
+
+	var validOSVersion bool
+	for _, option := range validOptions {
+		if osVersion == option {
+			validOSVersion = true
+		}
+	}
+
+	if !validOSVersion {
+		return errors.Errorf("Invalid Windows Server OS Version: %s. Valid OS version are: %s", osVersion, node.ValidWindowsOSVersions())
+	}
+
+	return nil
```

Comment:
```suggestion
	if validOptions[osVersion] {
		return nil
	}

	return errors.Errorf("Invalid Windows Server OS Version: %s. Valid OS version are: %s", osVersion, node.ValidWindowsOSVersions())
```

Using the map mentioned above greatly simplifies this